BackofficeBundle/Resources/public/smartadmin/js/app.js

```diff
@@ -762,6 +762,11 @@ $.fn.extend({
 				} // end else
 			} // end if
 		});
+
+		// prevent url load on extends accordion menu by click on collapse sign
+		$this.find(".collapse-sign").on("click", function(event) {

```

Comment:
I suggest to use a selector argument to restrict the listening to the nav element